spec/models/time_entry_spec.rb

```diff
@@ -351,5 +351,76 @@
         it { expect(TimeEntry.visible(time_entry2.user, project).all).to match_array([time_entry2]) }
       end
     end
+
+    context 'calculate dates' do
+      let(:user3) { FactoryGirl.create(:user) }
+      let(:late_time_entry) do
+        FactoryGirl.create(:time_entry,
+                           project: project,
+                           work_package: work_package,
+                           spent_on: 2.days.ago,
+                           hours: hours,
+                           user: user3,
+                           rate: hourly_one,
+                           comments: 'ipsum')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.